docs/releases/breaking-changes.md

```diff
@@ -11,6 +11,39 @@ these to a minimum, but rest assured we only make them with good reason.
 
 Starting with Directus 10.0, here is a list of potential breaking changes with remedial action you may need to take.
 
+## Version 10.6.2
+
+### Swapping Parameters and Auth Mode for Refresh Method in the SDK
```

Comment:
Oversight from my side: We should use past tense as it's the case with the existing titles
```suggestion
### Swapped Parameters and Auth Mode for Refresh Method in the SDK
```